app/controlplane/internal/conf/controlplane/config/v1/conf.proto

```diff
@@ -65,6 +65,16 @@ message Bootstrap {
 
   // Configuration to enable Prometheus integration for the specified organizations
   repeated PrometheusIntegrationSpec prometheus_integration = 11;
+
+  // providers for external policy repositories
+  repeated PolicyProvider policy_providers = 12;
+}
+
+message PolicyProvider {
+  string name = 1 [(buf.validate.field).string.min_len = 1];
```

Comment:
we should probably add some validations to these names since they will be used as prefixes. We could use the same validation we use for other names.